libraries-2/src/test/java/com/baeldung/parallel_collectors/ParallelCollectorsUnitTest.java

```diff
@@ -40,7 +40,7 @@ public void shouldProcessInParallelWithParallelCollectors() {
         List<Integer> ids = Arrays.asList(1, 2, 3);
 
         CompletableFuture<List<String>> results = ids.stream()
-          .collect(parallelToList(i -> fetchById(i), executor, 4));
+          .collect(parallelToList(ParallelCollectorsUnitTest::fetchById, executor, 4));
```

Comment:
is this an improvement or a fix?

Reply:
Yes this is an improvement to is the new way to call a static method in a lambda expression.